### DISCUSSION TITLE
PhotoPrism lxc configuration folder

### DISCUSSION THREAD

**Discussion by gianlucaf81** [tteck/Proxmox#2574] [Show and tell]
I lost a lot of time to understand that the folder to be indexed was inside `opt/photoprism/photos/originals/`, 
It would be useful to write it in the notes. 
However this is the configuration in the `etc/pve/lxc/XXX.conf` file of the lxc

`mp0: /mnt/pve/yourmontedfolder/,mp=/opt/photoprism/photos/originals/`

I apologize if this is trivial for many but I found conflicting information on the location of this folder and in fact it didn't work until I discovered it.

**Comment by tteck** [comment]
It's easily discovered if you look at the code
https://github.com/tteck/Proxmox/blob/d77cffd0068d2a60b03fd620c8fbf39af558bef7/install/photoprism-install.sh#L32

**Comment by gianlucaf81** [comment]
Do you have experience with "no space left on device"? I increased the size of the lxc (from 8 to 17GB) but it doesn't seem to want to start again. Thank you

**Reply by Narkowii** [reply]
not for me, but my LXC size is 60GB...

**Reply by gianlucaf81** [reply]
OMG... the db requires more space than I thought.

**Comment by gianlucaf81** [comment]
Again  PhotoPrism, I would like to move the "storage" folder to the NAS but I get this error, I can't understand what it is. thanks

`x photoprism.service - PhotoPrism service
     Loaded: loaded (/etc/systemd/system/photoprism.service; enabled; preset: enabled)
     Active: failed (Result: exit-code) since Tue 2024-03-05 10:23:55 CET; 5s ago
    Process: 144 ExecStart=/opt/photoprism/bin/photoprism up -d (code=exited, status=1/FAILURE)
        CPU: 2.206s

Mar 05 10:23:33 pprims-gian systemd[1]: Starting photoprism.service - PhotoPrism service...
Mar 05 10:23:35 pprims-gian photoprism[144]: time="2024-03-05T10:23:35+01:00" level=info msg="config: case-insensitive file system detected"
Mar 05 10:23:35 pprims-gian photoprism[144]: time="2024-03-05T10:23:35+01:00" level=info msg="Become a member today, support our mission and enjoy our member benefits! <F0><9F><92><>
Mar 05 10:23:35 pprims-gian photoprism[144]: time="2024-03-05T10:23:35+01:00" level=info msg="Visit https://www.photoprism.app/membership to learn more."
Mar 05 10:23:55 pprims-gian photoprism[144]: time="2024-03-05T10:23:55+01:00" level=warning msg="config: database is locked (migrate)"
Mar 05 10:23:55 pprims-gian photoprism[144]: time="2024-03-05T10:23:55+01:00" level=fatal msg="failed writing process id to /mnt/photoprism/storage/photoprism.pid"
Mar 05 10:23:55 pprims-gian systemd[1]: photoprism.service: Control process exited, code=exited, status=1/FAILURE
Mar 05 10:23:55 pprims-gian systemd[1]: photoprism.service: Failed with result 'exit-code'.
Mar 05 10:23:55 pprims-gian systemd[1]: Failed to start photoprism.service - PhotoPrism service.
Mar 05 10:23:55 pprims-gian systemd[1]: photoprism.service: Consumed 2.206s CPU time.`

![Senza-titolo-1](https://github.com/tteck/Proxmox/assets/39622729/e49e282b-db42-4887-af5c-57eaaef07df4)

**Reply by gianlucaf81** [reply]
Clarification: the files are written correctly to the shared folder on the NAS, however I get that error and Photoprism doesn't start.

**Reply by DrDonoso** [reply]
I was facing the same error and I just solved by changing the pid file location with PHOTOPRISM_PID_FILENAME=/opt/photoprism/photoprism.pid. Check reference [here](https://docs.photoprism.app/getting-started/config-options/#daemon-mode)

**Reply by gianlucaf81** [reply]
So did you just add that line or did you change the location of the pid file?
